.github/workflows/CI.yml

```diff
@@ -58,8 +57,7 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        python -m pip install flake8 pytest
-        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+        if [ -f requirements_dev.txt ]; then pip install -r requirements_dev.txt; fi
```

Comment:
```suggestion
      pip install -r requirements_dev.txt
```

Reply:
The bash syntax does not work for the Windows build. But I don't know what the Windows shell equivalent is.

Reply:
I think I fixed; removed the if statement everywhere